### ISSUE TITLE
Remove reference to openssl

### ISSUE THREAD

**Issue by bombarie** [mattfelsen/ofxMyoWebSockets#8]
I was getting a build error with your example project because it's looking for these files in [ofxLibwebsockets](https://github.com/labatrockwell/ofxLibwebsockets)

````
../../../addons/ofxLibwebsockets/libs/openssl/lib/osx/libcrypto.a
../../../addons/ofxLibwebsockets/libs/openssl/lib/osx/libssl.a
````

Turns out oF has openssl built in now and the ofxLibwebsockets guy removed these now unnecessary files from his project. Maybe you could remove them from your example project to prevent confusion, thanks!

**Comment by mattfelsen** [comment]
Hi @bombarie, thanks for the heads up! Indeed, @robotconscience removed openssl and I've been behind the curve in getting my projects up to date. I just pushed fixes to both the master & develop branches.

For what it's worth, you should check out the develop branch. I really should put together some better examples, but the dev branch has some more features and is more current with the newer SDKs from Thalmic. Let me know if you run into trouble or have any questions – I'll do my best to help!

**Comment by mattfelsen** [comment]
Fixed via b2a3190786b8cb0e55000f9ea5d80524d8e05fb3

**Comment by bombarie** [comment]
Tnx for the fix @mattfelsen, I'll check out the devbranch too. Cheers!
